d2bs/kolbot/libs/common/Misc.js

```diff
@@ -28,7 +28,6 @@ var Skill = {
 		case 112: // Blessed Hammer
 		case 116: // Conversion
 		case 126: // Bash
-		case 130: // Howl
```

Comment:
why is this deleted?

Reply:
it's a duplicate,130/howl case is defined further down with specific calculations